test/e2e/linuxbridge_test.go

```diff
@@ -0,0 +1,102 @@
+package e2e
+
+import (
+	"context"
+	"io/ioutil"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/types"
+	yaml "sigs.k8s.io/yaml"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+
+	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1"
+)
+
+var (
+	retryInterval        = time.Second * 5
+	timeout              = time.Second * 60
+	cleanupRetryInterval = time.Second * 1
+	cleanupTimeout       = time.Second * 5
+	createBridgeCr       = "deploy/crds/create-br1-linux-bridge.yaml"
+	deleteBridgeCr       = "deploy/crds/delete-br1-linux-bridge.yaml"
+	firstNodeName        = "node01" // TODO: Get it from cluster
+	bridgeName           = "br1"
+	namespace            string
+	ctx                  *framework.TestCtx
+)
+
+var _ = Describe("Linux Bridge", func() {
+	Context("when created", func() {
+
+		var (
+			state nmstatev1.NodeNetworkState
+		)
+
+		BeforeEach(func() {
```

Comment:
this block all the way to line 52 will be needed in all tests. would you mind taking a look at https://github.com/kubevirt/cluster-network-addons-operator/pull/118/commits/7120eed711981fd5c565e77c5028ba7934c10288#diff-e8664074431f2653c7da0fc9019a1935R22, we can use the same logic here.

Reply:
Done